src/contentmanager.cpp

```diff
@@ -189,6 +188,22 @@ QString ContentManager::downloadBook(const QString &id)
     return QString::fromStdString(download->getDid());
 }
 
+void ContentManager::eraseBook(const QString& id)
+{
+    kiwix::Book book = mp_library->getBookById(id);
+    QString dirName = QString::fromUtf8(getDataDirectory().c_str());
+    QString fileSelection = QString::fromUtf8(book.getPath().c_str()).remove(0, getDataDirectory().size() + 1) + "*";
```

Comment:
We should be careful here.
If a user open a zim file in its own directory, the file is added to the library but it is not copied in the data directory.
Just removing the size of the data directory from the file path may lead to strange results.
Maybe we could use `getLastPathElement` from kiwix-lib to get the filename.
If the file is not in the data directory, the `dir.entryList()` will be empty and we will remove nothing. And so it should be ok.